pkgs/games/keeperrl/default.nix

```diff
@@ -0,0 +1,57 @@
+{ stdenv, fetchFromGitHub, SDL2, SDL2_image, openal, libvorbis, zlib, curl,
+  libGL, makeWrapper, boost, makeDesktopItem }:
+
+stdenv.mkDerivation rec {
+  pname = "keeperrl";
+  version = "2019-10-21";
+
+  src = fetchFromGitHub {
+    owner = "miki151";
+    repo = "keeperrl";
+    rev = "74b89ea";
```

Comment:
Use the full revision, not a shorthand form, because shorthand forms can become ambiguous